sdk.sln

```diff
@@ -262,6 +262,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.NET.Sdk.WorkloadM
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver", "src\Resolvers\Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver\Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.csproj", "{56F189B0-6A8E-4A9D-B7AE-24241204E98D}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tools", "Tools", "{71A9F549-0EB6-41F9-BC16-4A6C5007FC91}"
```

Comment:
call it BuiltInTools? It is too general

Reply:
`GlobalTools`? I'm fine with either.

Reply:
They are not really global tools (shared by many SDKs). I prefer BuiltInTools